packages/settings/src/settings.ts

```diff
@@ -63,6 +65,7 @@ export interface ISettings
  */
 export const settings: ISettings = {
 
+    ADAPTER: BrowserAdapter,
```

Comment:
Would be good to add some docs here for this property, IAdapter and the Node and Browser adapters.